### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by TwilightDuck** [yoziru/esphome-tesla-ble#15]
Hi,

Just wanted to let you know that I'll be trying this out. Just ordered my first ESP32 device ever. 
Excited!

**Comment by amit-nz** [comment]
+1 on this, thanks a ton to @yoziru for building this! It will be tremendously helpful after the API changes earlier this year that requires a bunch of infrastructure just to be able to send 50 commands to the API per day.

**Comment by Bre77** [comment]
Yeah I'll add to this, I'm the code owner of Tessie, Teslemetry, and as of 2024.8 Tesla Fleet, and I expected I'd need to write this in python in Home Assistant and use Bluetooth proxies, but this achieves the same thing! I might still add this capability to Home Assistant directly, but having your code as reference is going to be awesome.

**Comment by madsdyd** [comment]
@Bre77 Oh, please add it directly to Home Assistant so us mortal peeps can leverage it :-D

**Comment by mrd0n** [comment]
Just jumping in to say Thank You!   

I am up and running with a WEMOS D1 Mini using the Generic build, adopted in ESPHome within Home assistant and managing car charging with excess solar using a native HA automation. Amazing!

**Comment by anthonykeane** [comment]
Great stuff, looking fwd to trying, thanks Heaps

**Comment by yoziru** [comment]
Hope it worked out! Closing.
